android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -466,6 +469,10 @@ public void clear() {
       mUrlToDeviceAddress.clear();
       notifyDataSetChanged();
     }
+
+    public void redrawListView() {
```

Comment:
1. By [definition notifyDataSetChanged](http://developer.android.com/reference/android/widget/BaseAdapter.html#notifyDataSetChanged()) implies that the listview should be refreshed. Why not call super.notifyDataSetChanged() directly, instead of calling it through a another function that just calls that one function?